lib/ansible/parsing/mod_args.py

```diff
@@ -182,7 +182,7 @@ def _normalize_parameters(self, thing, action=None, additional_args=None):
             if args and 'args' in args:
                 tmp_args = args.pop('args')
                 if isinstance(tmp_args, string_types):
-                    tmp_args = parse_kv(tmp_args)
+                    tmp_args = parse_kv(tmp_args, check_raw=bool(action in FREEFORM_ACTIONS))
```

Comment:
What's this change for? I noticed it was in the original commit, along with changelog entries that appear to be related, but those changelog entries were removed in a later commit.

Reply:
Also, a `__contains__()` check already returns a `bool`. No need to convert bool to bool.
```suggestion
                    tmp_args = parse_kv(tmp_args, check_raw=action in FREEFORM_ACTIONS)
```